requirements/edx/github.txt

```diff
@@ -63,8 +63,10 @@ git+https://github.com/mitocw/django-cas.git@60a5b8e5a62e63e0d5d224a87f0b489201a
 git+https://github.com/pmitros/pyfs.git@96e1922348bfe6d99201b9512a9ed946c87b7e0b
 # The officially released version of django-debug-toolbar-mongo doesn't support DJDT 1.x. This commit does.
 git+https://github.com/hmarr/django-debug-toolbar-mongo.git@b0686a76f1ce3532088c4aee6e76b9abe61cc808
-# custom opaque-key implementations for ccx
--e git+https://github.com/jazkarta/ccx-keys.git@e6b03704b1bb97c1d2f31301ecb4e3a687c536ea#egg=ccx-keys
+
+# custom opaque-key implementations for CCX
+git+https://github.com/edx/ccx-keys.git@clintonb/updates#egg=ccx-keys==0.1.1
```

Comment:
This will be changed to use the tag `0.1.1` after we get a successful build of https://build.testeng.edx.org/view/packer/job/build-packer-ami/313/console and merge edx/ccx-keys/pull/1.